MiBench/automotive/bitcount/src/bitcnts.c

```diff
@@ -52,42 +46,45 @@ int main(int argc, char *argv[])
     "Non-recursive bit count by bytes (AR)",
     "Shift and count bits"
   };
-  if (argc<2) {
-    fprintf(stderr,"Usage: bitcnts <iterations>\n");
+  if (argc < 2) {
+    fprintf(stderr, "Usage: bitcnts <iterations>\n");
     exit(-1);
-	}
-  iterations=atoi(argv[1]);
-  
+  }
```

Comment:
It's difficult to review your pull request when you change the formatting of the code that you do not add or change. Please try not to do that.

Reply:
Okay